python/envs/ControlRL/ControlHVACEnv.py

```diff
@@ -72,6 +71,11 @@ def __init__(self):
         self.action_range = cfg['action_range']
         #Action space is the up & down range for the 5 actions 
         self.action_space = spaces.Box(-self.action_range, self.action_range, shape=(self.num_ins,), dtype=np.float32)  # set to +/- 5
+
+        # initialize logging paramteres
+        self.episodes=0
+        self.data=''
+
         self.seed()
 
     def get_funcs(self,var):
```

Comment:
style comment: in function names, try to put a space between parameters i.e. `get_funcs(self, var)`